sdk/network/arm-network/README.md

```diff
@@ -6,7 +6,7 @@ Network Client
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/network/arm-network) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/arm-network) |
-[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-network) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-network?view=azure-node-preview) |
```

Comment:
Should we change it back to the stable version reference ? @dw511214992 @sarangan12